string/punctuation.js

```diff
@@ -68,6 +69,12 @@ const languageSpecificRules = {
 	ja: [ // Japanese
 		[/(?<=[^\p{L}\d]|^)-(.+?)-(?=[^\p{L}\d]|$)/ug, '–$1–'], // dashes used as brackets
 	],
+	he: [ // Hebrew
+		/* apostrophes */
+		[/’/g, '׳'],
```

Comment:
This won't work, it will also convert all Hebrew single quotes into apostrophes.

Ideally every rule should convert ASCII to Unicode to avoid such mistakes, which means that the general rule for apostrophes has to be more strict.
Currently it just takes all remaining ASCII apostrophes and converts them, this is something which I wanted to improve for a long time, but never got around to do 😅